lib/logger/lib/logger/utils.ex

```diff
@@ -84,22 +84,37 @@ defmodule Logger.Utils do
   Receives a format string and arguments and replace `~p`,
   `~P`, `~w` and `~W` by its inspected variants.
   """
-  def inspect(format, args, truncate, opts \\ %Inspect.Opts{})
+  def inspect(format, args, truncate, opts \\ %Inspect.Opts{}) do
```

Comment:
I've left it here for backward compatibility. Will we deprecate it?

Reply:
This module has `@moduledoc false`, so please remove it!

Reply:
PURRRFECT! 🐈